src/DataReader/parse_queries.rs

```diff
@@ -16,7 +16,7 @@ pub struct QueryParser;
 ///For clarification see documentation on pest crate
 
 pub fn parse_to_query(query: &str) -> Vec<Query> {
-    let queries = parse_to_expression_tree(query);
+    let queries = parse_to_expression_tree(query).unwrap();
```

Comment:
Error?

Reply:
@t-lohse this is a problem for the "cause of failure" group. Sebastian said we could just leave them be for now.